test/everypolitician/popolo/event_test.rb

```diff
@@ -27,4 +31,24 @@ def test_accessing_event_properties
     assert_equal '2014-11-27', event.start_date
     assert_nil event.end_date
   end
+
+  def test_event_people_returns_array_of_person_objects
+    event = popolo_wales.events.map { |e| e }[4]
```

Comment:
Mapping every element to itself seems remarkably odd, and almost certainly wrong. At the very least you should be calling `.to_a`, but it's unclear why you want to be operating on element `[4]` anyway. Wouldn't `popolo_wales.events.first` suffice?